pkg/reconciler/topology/partitionset/resources.go

```diff
@@ -48,3 +50,10 @@ func generatePartition(name string, matchExpressions []metav1.LabelSelectorRequi
 		},
 	}
 }
+
+func min(a, b int) int {
```

Comment:
IMHO we don't need to accumulate this sort of logic. Just inline it.

Reply:
inlining is done by the compiler. I find this way more readable and it seems that it is not an uncommon practice.